var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -139,6 +183,14 @@ def edit(self, spec, prefix):
 
             filter_file("^CFLAGS[ ]{0,}=.*$", "CFLAGS ?=", "makefile.include")
 
+        if "+fftlib" in spec:
+            filter_file("^#FCL[ ]{0,}=fftlib.o", "FCL += fftlib/fftlib.o", "makefile.include")
+            filter_file("^#CXX_FFTLIB", "CXX_FFTLIB", "makefile.include")
+            filter_file("^#INCS_FFTLIB", "INCS_FFTLIB", "makefile.include")
+            filter_file("^#LIBS", "LIBS", "makefile.include")
+            filter_file(
+                "LIBS[ ]{0,}=.*$", "LIBS=-lstdc++ fftlib/fftlib.o -ldl", "makefile.include"
```

Comment:
(no change needed) I'm surprised when users have to add low-level link references like `-ldl` and `-lstdc++`, so I'd be curious if this came naturally (i.e. mirrors other similar changes in the file). I don't need to hold up the PR on it though.